packages/ui/primitives/document-flow/add-subject.tsx

```diff
@@ -105,7 +131,7 @@ export const AddSubjectFormPartial = ({
                 </li>
                 <li className="text-muted-foreground">
                   <code className="text-muted-foreground bg-muted-foreground/20 rounded p-1 text-sm">
-                    {'{signer.email}'}
+                    {'{signer.meta}'}
```

Comment:
`signer.email`

Reply:
The `Cmd+Shift+F` has caught up to me, lol